pkg/schedulingtypes/replicascheduler.go

```diff
@@ -178,7 +210,12 @@ func (s *ReplicaScheduler) Reconcile(obj pkgruntime.Object, qualifiedName Qualif
 		return StatusNeedsRecheck
 	}
 
-	if !s.plugins[kind].TemplateExists(qualifiedName.String()) {
+	v, ok := s.plugins[kind]
```

Comment:
Please prefer a descriptive variable name to a single-character names where possible.  Acceptable uses include loop indexes and receiver variables (e.g. `s` in this function)